hack/.argocdUtils

```diff
@@ -9,3 +9,48 @@ argocdAddCluster() {
     setLabel argocd.argoproj.io/secret-type cluster | 
     kubectl apply --context kind-${hubCluster} -f -
 }
+
+
+CreateAgentSecret() {
```

Comment:
This isn't used anywhere that I can see.
Also, it doesn't seem to be argocd related

Reply:
oh yeah, this an artifact from rebasing with Sergio's changes. I'll remove it.